lib/tasks/complete_technical_failure_emails.rake

```diff
@@ -0,0 +1,9 @@
+namespace :complete_technical_failure_emails do
+  desc "Update emails marked as technical failure to have a completed_at time"
+  task fill_completed_at_time: :environment do
+    incomplete_technical_failure_emails = Email.where(failure_reason: "technical_failure").where(finished_sending_at: nil)
+    incomplete_technical_failure_emails.each do |email|
+      email.update(finished_sending_at: DateTime.now)
```

Comment:
Although we don't have enough information to get the exact date, I wonder if we can use the `updated_at` of the delivery attempt which will be closer to the time that this happened.

Something like:

```suggestion
      email.update(finished_sending_at: email.delivery_attempts.last.created_at)
```

We could add `.includes(:delivery_attempt)` on the query above to avoid an extra query here.